support/build/Makefile.rules

```diff
@@ -873,10 +873,6 @@ $(call libname2preolib,$(1)): $$($(call vprefix_lib,$(1),OBJS)) \
 			      $$($(call vprefix_lib,$(1),OBJS-y)) \
 			      $$(EACHOLIB_OBJS) \
 			      $$(EACHOLIB_OBJS-y) \
-			      $$($(call vprefix_lib,$(1),ALIBS)) \
-			      $$($(call vprefix_lib,$(1),ALIBS-y)) \
-			      $$(EACHOLIB_ALIBS) \
-			      $$(EACHOLIB_ALIBS-y) \
```

Comment:
Hum... 🤔 either this commits needs a different description why we remove the ALIB option for libraries or the patch should not be part of the PR?

Reply:
Got it, yeah, I've removed this commit entirely